### PR TITLE
Added test coverage with codecov.io

### PR DESCRIPTION
**Pull request by j-bennet**
Link to review coverage:

https://codecov.io/github/dbcli

(login with github).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,14 @@ python:
   - "3.4"
 
 install: 
-  - pip install . pytest mock
+  - pip install codecov
```
Comment (amjith) [inline .travis.yml]: Why was pytest and mock removed from the `pip install`?

**Comment by j-bennet** [comment]
Forgot that they're not in setup.py. How does the build not fail? Interesting. I'll add them back.

**Comment by amjith** [comment]
Yeah I was wondering the same thing. It didn't fail because travis ci installs them by default. 

http://docs.travis-ci.com/user/languages/python/#Pre-installed-packages

But I think it is better to install them explicitly.

**Comment by j-bennet** [comment]
Added them back.

**Comment by amjith** [comment]
Looks good. Thanks for taking the time to add it in. 

Closes #295. 

:mailbox:

**Pull request merged**